webapp/packages/plugin-app-logo/src/PluginBootstrap.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { Bootstrap, injectable } from '@cloudbeaver/core-di';
+import { TopNavService } from '@cloudbeaver/plugin-top-app-bar';
+
+import { Logo } from './Logo.js';
```

Comment:
please use `importLazyComponent` - so main goal here is to initialize "business" logic as fast as possible and then load UI part so every time you import components in services use `importLazyComponent`